### DISCUSSION TITLE
Welcome to Mistful Crimson Morning Discussions!

### DISCUSSION THREAD

**Discussion by pahaze** [pahaze/Mistful-Crimson-Morning#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

Remember to be civil and have fun here.

**Comment by SevoStar** [comment]
Wow, seems like im the only one here with a question interestingly enough
![image](https://github.com/pahaze/Mistful-Crimson-Morning/assets/144753006/c29d4cc6-e8ae-4091-b793-1b15d44fa36b)
![image](https://github.com/pahaze/Mistful-Crimson-Morning/assets/144753006/e91f6a79-3faf-49ad-b94d-b8392be118a7)

Hi, I was wondering if I could know on how to put the VCR Overlay from the main week into other songs.
I got bored earlier and recharted Misery and Mist (Older) and want to put the overlay in those songs for authenticity but can't seem to figure out how. Is it possible by any chance that you know how it was done? Thanks in advance!
(If you can't figure it out either, then that's ok!)

**Reply by pahaze** [reply]
Oh yeah, absolutely! To add the VCR shaders to any song, in PlayState, there's a line where you can define what songs have it.

https://github.com/MistThrowaway/Mistful-Crimson-Morning/blob/master/source/PlayState.hx#L1543

To add another song, add a `|` and put your song in as a string... For example, where you're playing Served here, you would change

`case "doomsday" |"humiliation" | "mist": // Doomsday, Humilation, Mist`

to

`case "doomsday" |"humiliation" | "mist" | "served": // Doomsday, Humilation, Mist, Served`

It needs to be lowercase as the switch case checks the song's lowercase name! Hope this helps!

**Reply by SevoStar** [reply]
Awesome, thanks so much! Honestly I wasn't expecting a reply considering how dead this mod and thread was, but the help is appreciated! Have a good day or night! You've earned it :]

**Reply by pahaze** [reply]
> Awesome, thanks so much! Honestly I wasn't expecting a reply considering how dead this mod and thread was, but the help is appreciated! Have a good day or night! You've earned it :]

You're welcome! I'd update the mod, but there's nothing to update it with honestly, minus some better memory management for web builds, lol. I've just been busy with IRL stuff... and I'm lazy when I'm not dealing with anything 😂

If you need any more help with anything, don't hesitate to ask! You also have a good day/night!
